eras/shelley/impl/testlib/Test/Cardano/Ledger/Shelley/Imp/UtxoSpec.hs

```diff
@@ -20,10 +21,13 @@ spec ::
   ) =>
   SpecWith (ImpTestState era)
 spec = describe "UTXO" $ do
+  it "sendCoinTo" $ do
+    addr <- freshKeyAddr_
+    void $ sendCoinTo addr (Coin 999833135)
```

Comment:
It was a temporary test that I was using it to implement MinUTxO fixup. It is no longer needed, since as you very well pointed out, there is little value in having it.